src/docs/sdk/event-payloads/contexts.mdx

```diff
@@ -273,6 +273,14 @@ output of the `uname` syscall.
 system. For some well-known runtimes, Sentry will attempt to parse `name` and
 `version` from this string, if they are not explicitly given.
 
+`distribution`
+
+: _Optional_. An object that provides meta-data for Linux distributions. Contains the following keys:
+
+- `name`: A name that is stable for each distribution (examples: `ubuntu`, `rhel`, `alpine`).
```

Comment:
Is there an exhaustive list of identifiers we can link to?

Reply:
The official documentation contains most (all?) of the identifiers: https://www.freedesktop.org/software/systemd/man/latest/os-release.html.

I added [test data](https://github.com/getsentry/sentry-native/tree/c18b5ff8953e602af4cf603583e5b62f384a613f/tests/fixtures/os_releases) to the sentry-native repo, of which I can quickly gather an exhaustive list from a representative subset (not to link to, but to compare with the docs).

It would be a good idea to link the `os-release` docs from `develop`, but then it makes sense to show the current mapping, too, right? I think it boils down to whether we want to use their identifiers. 

If we use them, we can say that whatever is in `distribution` is a straight translation from `/etc/os-release`; we link to the official docs, and everyone can look them up or request the addition of new ones.

With any kind of mapping, we can leave it open if we gather distro-related data from other sources in the future, but we need to specify where data comes from individually for each entry.

Initially, I was in the latter camp because the most valuable identifiers might have less obvious names for people not familiar with `/etc/os-release`. But now I think that introducing our own identifier mapping could make things even less obvious.

Reply:
I think as long as we're compatible with the "official" identifiers we should link to them, once we diverge either document it here or update the link to where one can see the differences/list of values we use.

Reply:
Oh yeah sorry, I meant the names like `ubuntu`, `rhel`, `alpine`. Didn't realize identifiers is ambiguous.